ntbs-service-unit-tests/Services/UserSearchServiceTest.cs

```diff
@@ -190,31 +190,7 @@ public async Task OrderQueryableAsync_DoesNotReturnInactiveFamilyNameMatchingUse
             Assert.Equal(1, results.Count);
             Assert.Equal(activeCaseManager, results.Single());
         }
-        
-        [Fact]
-        public async Task OrderQueryableAsync_ReturnsRegionMatchingCaseManagersAndUsers()
```

Comment:
Rather than delete these tests, wouldn't it be better to invert them i.e. change the expected outcomes? I think we should have at least one test that's checking that users aren't included in results when their TB service/region matches the search term